fix: Incorrect Content-Type Expectation for organization api endpoints

JonasDev17 opened the pull request:
Simply changed the media type from  `APPLICATION_JSON` to `TEXT_PLAIN`

Closes #34401.

edewit committed:
added description to denied consent and show on ErrorPage

fixes: #28328
Signed-off-by: Erik Jan de Wit <erikjan.dewit@gmail.com>

vramik committed:
Declining terms and conditions in account-console results in error

Closes #28328

Signed-off-by: vramik <vramik@redhat.com>

edewit committed:
* hide standard text when we have a description
* lookup description in message bundle

Signed-off-by: Erik Jan de Wit <erikjan.dewit@gmail.com>

edewit committed:
added message

Signed-off-by: Erik Jan de Wit <erikjan.dewit@gmail.com>

edewit committed:
always try and translate the error message

Signed-off-by: Erik Jan de Wit <erikjan.dewit@gmail.com>

graziang committed:
Fix flaky test for concurrent client creation on H2 database

Closes #29290

Signed-off-by: Giuseppe Graziano <g.graziano94@gmail.com>

rmartinc committed:
Set client in context for dynamic scopes calculation
Closes #33684

Signed-off-by: rmartinc <rmartinc@redhat.com>

JonasDev17 committed:
Update OrganizationMembersResource.java

Signed-off-by: Jonas Suter <jonas_suter@gmx.ch>

JonasDev17 committed:
Closes #34401

sguilhen committed:
Ensure searched LDAPObject is properly cached before other methods that trigger user validation run

Closes #34050

Signed-off-by: Stefan Guilhen <sguilhen@redhat.com>

TheOignon committed:
Implement pagination for getLDAPRoleMappings (#34043)

* Implement pagination for getLDAPRoleMappings

On Active Directory, allow to retrieve more groups than the MaxPageSize
(default to 1000). Without this patch, we need to increase the
MaxPageSize which does not really scale. Implemented only for the
LoadRolesByMember startegy.

Closes #34042

Signed-off-by: Alexi Vandevoorde <alexi@vandevoor.de>

jonkoops committed:
Disable major updates for PatternFly (#34390)

Closes #34389

Signed-off-by: Jon Koops <jonkoops@gmail.com>

sguilhen committed:
Check if LDAPObject is available from a previously cached proxied user

Closes #34412

Signed-off-by: Stefan Guilhen <sguilhen@redhat.com>

dependabot[bot] committed:
Bump @types/node from 22.7.9 to 22.8.2 (#34413)

Bumps [@types/node](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/node) from 22.7.9 to 22.8.2.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/node)

---
updated-dependencies:
- dependency-name: "@types/node"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump cypress-split from 1.24.4 to 1.24.5 (#34414)

Bumps [cypress-split](https://github.com/bahmutov/cypress-split) from 1.24.4 to 1.24.5.
- [Release notes](https://github.com/bahmutov/cypress-split/releases)
- [Commits](https://github.com/bahmutov/cypress-split/compare/v1.24.4...v1.24.5)

---
updated-dependencies:
- dependency-name: cypress-split
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump eslint-plugin-react-compiler (#34416)

Bumps [eslint-plugin-react-compiler](https://github.com/facebook/react/tree/HEAD/compiler/packages/eslint-plugin-react-compiler) from 19.0.0-beta-8a03594-20241020 to 19.0.0-beta-9ee70a1-20241017.
- [Release notes](https://github.com/facebook/react/releases)
- [Changelog](https://github.com/facebook/react/blob/main/CHANGELOG.md)
- [Commits](https://github.com/facebook/react/commits/HEAD/compiler/packages/eslint-plugin-react-compiler)

---
updated-dependencies:
- dependency-name: eslint-plugin-react-compiler
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump @eslint/compat from 1.2.1 to 1.2.2 (#34418)

Bumps [@eslint/compat](https://github.com/eslint/rewrite) from 1.2.1 to 1.2.2.
- [Release notes](https://github.com/eslint/rewrite/releases)
- [Changelog](https://github.com/eslint/rewrite/blob/main/release-please-config.json)
- [Commits](https://github.com/eslint/rewrite/compare/compat-v1.2.1...compat-v1.2.2)

---
updated-dependencies:
- dependency-name: "@eslint/compat"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump @types/chai from 5.0.0 to 5.0.1 (#34417)

Bumps [@types/chai](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/chai) from 5.0.0 to 5.0.1.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/chai)

---
updated-dependencies:
- dependency-name: "@types/chai"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump typescript-eslint from 8.11.0 to 8.12.0 (#34423)

Bumps [typescript-eslint](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/typescript-eslint) from 8.11.0 to 8.12.0.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/typescript-eslint/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v8.12.0/packages/typescript-eslint)

---
updated-dependencies:
- dependency-name: typescript-eslint
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump @patternfly/react-core from 5.4.2 to 5.4.8 (#34419)

Bumps [@patternfly/react-core](https://github.com/patternfly/patternfly-react) from 5.4.2 to 5.4.8.
- [Release notes](https://github.com/patternfly/patternfly-react/releases)
- [Commits](https://github.com/patternfly/patternfly-react/compare/@patternfly/react-core@5.4.2...@patternfly/react-core@5.4.8)

---
updated-dependencies:
- dependency-name: "@patternfly/react-core"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

vramik committed:
Keep `error` and `error_description` query params in login url.

Signed-off-by: vramik <vramik@redhat.com>

evankanderson committed:
Adding Minder & Stacklok usage (#34357)

Signed-off-by: Evan Anderson <evan@stacklok.com>

edewit committed:
move library build to dist build (#34394)

towards: #34393

Signed-off-by: Erik Jan de Wit <erikjan.dewit@gmail.com>

JonasDev17 committed:
Merge branch 'keycloak:main' into main

edewit committed:
added missing message

Signed-off-by: Erik Jan de Wit <erikjan.dewit@gmail.com>

dependabot[bot] committed:
Bump i18next from 23.16.3 to 23.16.4 (#34378)

Bumps [i18next](https://github.com/i18next/i18next) from 23.16.3 to 23.16.4.
- [Release notes](https://github.com/i18next/i18next/releases)
- [Changelog](https://github.com/i18next/i18next/blob/master/CHANGELOG.md)
- [Commits](https://github.com/i18next/i18next/compare/v23.16.3...v23.16.4)

---
updated-dependencies:
- dependency-name: i18next
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump typescript-eslint from 8.12.0 to 8.12.1 (#34431)

Bumps [typescript-eslint](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/typescript-eslint) from 8.12.0 to 8.12.1.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/typescript-eslint/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v8.12.1/packages/typescript-eslint)

---
updated-dependencies:
- dependency-name: typescript-eslint
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump @patternfly/react-icons from 5.4.0 to 5.4.2 (#34421)

Bumps [@patternfly/react-icons](https://github.com/patternfly/patternfly-react) from 5.4.0 to 5.4.2.
- [Release notes](https://github.com/patternfly/patternfly-react/releases)
- [Commits](https://github.com/patternfly/patternfly-react/compare/@patternfly/react-icons@5.4.0...@patternfly/react-icons@5.4.2)

---
updated-dependencies:
- dependency-name: "@patternfly/react-icons"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump vitest from 2.1.3 to 2.1.4 (#34422)

Bumps [vitest](https://github.com/vitest-dev/vitest/tree/HEAD/packages/vitest) from 2.1.3 to 2.1.4.
- [Release notes](https://github.com/vitest-dev/vitest/releases)
- [Commits](https://github.com/vitest-dev/vitest/commits/v2.1.4/packages/vitest)

---
updated-dependencies:
- dependency-name: vitest
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

edewit committed:
some migrations have empty name (#34288)

fixes: #33309

Signed-off-by: Erik Jan de Wit <erikjan.dewit@gmail.com>

dependabot[bot] committed:
Bump uuid from 11.0.1 to 11.0.2 (#34420)

Bumps [uuid](https://github.com/uuidjs/uuid) from 11.0.1 to 11.0.2.
- [Release notes](https://github.com/uuidjs/uuid/releases)
- [Changelog](https://github.com/uuidjs/uuid/blob/main/CHANGELOG.md)
- [Commits](https://github.com/uuidjs/uuid/compare/v11.0.1...v11.0.2)

---
updated-dependencies:
- dependency-name: uuid
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

stianst committed:
Update node-cache/action.yml to use mvnw (#34438)

Signed-off-by: stianst <stianst@gmail.com>

agagancarczyk committed:
Added Memberships Modal (#33433)

* added MembershipsModal and fixed minor css issues

Signed-off-by: Agnieszka Gancarczyk <agagancarczyk@gmail.com>

* added test

Signed-off-by: Agnieszka Gancarczyk <agagancarczyk@gmail.com>

* improved test

Signed-off-by: Agnieszka Gancarczyk <agagancarczyk@gmail.com>

---------

Signed-off-by: Agnieszka Gancarczyk <agagancarczyk@gmail.com>

edewit committed:
added dummy script tag to prevent FOUC (#34329)

fixes: #33793

Signed-off-by: Erik Jan de Wit <erikjan.dewit@gmail.com>

edewit committed:
fixed key of policy group help (#34332)

fixes: #33618

Signed-off-by: Erik Jan de Wit <erikjan.dewit@gmail.com>

edewit committed:
added required attribute to multiline (#34336)

fixes: #32786

Signed-off-by: Erik Jan de Wit <erikjan.dewit@gmail.com>

edewit committed:
fix bundling issues (#34387)

fixes: #34386

Signed-off-by: Erik Jan de Wit <erikjan.dewit@gmail.com>

pruivo committed:
Update sizing guide for client credential grant

Closes #34347

Signed-off-by: Pedro Ruivo <pruivo@redhat.com>
Signed-off-by: Alexander Schwartz <aschwart@redhat.com>
Signed-off-by: Alexander Schwartz <alexander.schwartz@gmx.net>
Co-authored-by: Alexander Schwartz <aschwart@redhat.com>
Co-authored-by: Alexander Schwartz <alexander.schwartz@gmx.net>

BrunoSampaioDTx committed:
Use the response_permissions_limit value, if provided, to set the maximum number of results when retrieving resources by URI

Signed-off-by: BrunoSampaioDTx <bruno.sampaio@dtx-colab.pt>

pedroigor committed:
make sure error dialog is shown at the account console when declining terms

Signed-off-by: Pedro Igor <pigor.craveiro@gmail.com>

pedroigor committed:
Do not rely on the pwdLastSet attribute when updating AD entries

Closes #34467

Signed-off-by: Pedro Igor <pigor.craveiro@gmail.com>

vramik committed:
Fix error message in test

Signed-off-by: vramik <vramik@redhat.com>

thomasdarimont committed:
Revise help icons for WebauthnPolicy settings (#34465) (#34466)

- Add missing icons with help labels
- Use correct help text for webAuthnPolicyCreateTimeout

Fixes #34465

Signed-off-by: Thomas Darimont <thomas.darimont@googlemail.com>

isaac-mercieca committed:
fixes issue with labels not being applied for selected items defined as user profile attributes with type multiselect (#34457)

Signed-off-by: Isaac Mercieca <isaac.mercieca@rs2.com>

JonasDev17 committed:
Merge branch 'keycloak:main' into main

dependabot[bot] committed:
Bump @faker-js/faker from 9.0.3 to 9.1.0 (#34377)

* Bump @faker-js/faker from 9.0.3 to 9.1.0

Bumps [@faker-js/faker](https://github.com/faker-js/faker) from 9.0.3 to 9.1.0.
- [Release notes](https://github.com/faker-js/faker/releases)
- [Changelog](https://github.com/faker-js/faker/blob/next/CHANGELOG.md)
- [Commits](https://github.com/faker-js/faker/compare/v9.0.3...v9.1.0)

---
updated-dependencies:
- dependency-name: "@faker-js/faker"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

* removed use of deprecated userName()

Signed-off-by: Erik Jan de Wit <erikjan.dewit@gmail.com>

---------

Signed-off-by: dependabot[bot] <support@github.com>
Signed-off-by: Erik Jan de Wit <erikjan.dewit@gmail.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Erik Jan de Wit <erikjan.dewit@gmail.com>

dependabot[bot] committed:
Bump mocha from 10.7.3 to 10.8.1 (#34471)

Bumps [mocha](https://github.com/mochajs/mocha) from 10.7.3 to 10.8.1.
- [Release notes](https://github.com/mochajs/mocha/releases)
- [Changelog](https://github.com/mochajs/mocha/blob/main/CHANGELOG.md)
- [Commits](https://github.com/mochajs/mocha/compare/v10.7.3...v10.8.1)

---
updated-dependencies:
- dependency-name: mocha
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump typescript-eslint from 8.12.1 to 8.12.2 (#34469)

Bumps [typescript-eslint](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/typescript-eslint) from 8.12.1 to 8.12.2.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/typescript-eslint/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v8.12.2/packages/typescript-eslint)

---
updated-dependencies:
- dependency-name: typescript-eslint
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump @types/node from 22.8.2 to 22.8.4 (#34468)

Bumps [@types/node](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/node) from 22.8.2 to 22.8.4.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/node)

---
updated-dependencies:
- dependency-name: "@types/node"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump @patternfly/react-code-editor from 5.4.10 to 5.4.11 (#34472)

Bumps [@patternfly/react-code-editor](https://github.com/patternfly/patternfly-react) from 5.4.10 to 5.4.11.
- [Release notes](https://github.com/patternfly/patternfly-react/releases)
- [Commits](https://github.com/patternfly/patternfly-react/compare/@patternfly/react-code-editor@5.4.10...@patternfly/react-code-editor@5.4.11)

---
updated-dependencies:
- dependency-name: "@patternfly/react-code-editor"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump @playwright/test from 1.48.1 to 1.48.2 (#34372)

Bumps [@playwright/test](https://github.com/microsoft/playwright) from 1.48.1 to 1.48.2.
- [Release notes](https://github.com/microsoft/playwright/releases)
- [Commits](https://github.com/microsoft/playwright/compare/v1.48.1...v1.48.2)

---
updated-dependencies:
- dependency-name: "@playwright/test"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

ryanemerson committed:
Update caching docs to reflect that IP multicast is no longer used by default

Closes #34495

Signed-off-by: Ryan Emerson <remerson@redhat.com>
Signed-off-by: Alexander Schwartz <aschwart@redhat.com>
Co-authored-by: Alexander Schwartz <aschwart@redhat.com>

sguilhen committed:
Ensure hide_on_login has the default value set to 0 on MSSQL

Closes #34450

Signed-off-by: Stefan Guilhen <sguilhen@redhat.com>

edewit committed:
added table truncate on role description (#34289)

fixes: #32992

Signed-off-by: Erik Jan de Wit <erikjan.dewit@gmail.com>

JonasDev17 committed:
Update OrganizationIdentityProvidersResource.java

Signed-off-by: Jonas Suter <jonas_suter@gmx.ch>

JonasDev17 committed:
Merge branch 'main' into main

martin-kanis commented:
@JonasDev17 can you please squash commits into a single one and add `Closes #34401` into the commit message so the PR is linked with the issue?

dependabot[bot] committed:
Bump rollup from 4.24.2 to 4.24.3 (#34473)

Bumps [rollup](https://github.com/rollup/rollup) from 4.24.2 to 4.24.3.
- [Release notes](https://github.com/rollup/rollup/releases)
- [Changelog](https://github.com/rollup/rollup/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rollup/rollup/compare/v4.24.2...v4.24.3)

---
updated-dependencies:
- dependency-name: rollup
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

doucsag committed:
changed TEXT_PLAIN_UTF_8 to TEXT_PLAIN

weblate committed:
Updated translation for Japanese (#34448)

Language: ja

Update translation files

Updated by "Cleanup translation files" hook in Weblate.

Signed-off-by: Alexander Schwartz <alexander.schwartz@gmx.net>
Signed-off-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Alexander Schwartz <alexander.schwartz@gmx.net>

JonasDev17 committed:
Merge branch 'keycloak:main' into main

JonasDev17 commented:
> @JonasDev17 can you please squash commits into a single one and add `Closes #34401` into the commit message so the PR is linked with the issue?

@martin-kanis I believe the builld failed but I just updated the code from TEXT_PLAIN_UTF_8 to just TEXT_PLAIN. That might fix the build. However I do not know how to squash my commits and I can't seem to get it to work... should I create a new PR with a new branch?

martin-kanis commented:
> > @JonasDev17 can you please squash commits into a single one and add `Closes #34401` into the commit message so the PR is linked with the issue?
> 
> @martin-kanis I believe the builld failed but I just updated the code from TEXT_PLAIN_UTF_8 to just TEXT_PLAIN. That might fix the build. However I do not know how to squash my commits and I can't seem to get it to work... should I create a new PR with a new branch?

Yeah feel free to create a new PR. Also, don't forget to use `git commit -s ...` so your commit is signed.

doucsag committed:
Merge branch 'main' of https://github.com/JonasDev17/keycloak
